config/profile/rhdh/manager.yaml

```diff
@@ -71,7 +71,10 @@ spec:
       - command:
         - /manager
         args:
-        - --leader-elect
+          - --health-probe-bind-address=:8081
```

Comment:
"probe-bind-address" ?

Reply:
Same arguments as in https://github.com/redhat-developer/rhdh-operator/pull/338#discussion_r1803777201 to keep "health" in the name.
Also, `health-probe-bind-address` seems consistent with what's there in the manager `Options` struct: https://github.com/kubernetes-sigs/controller-runtime/blob/main/pkg/manager/manager.go#L226-L229